workflow/stitch.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# the following docker image requires github authentication
+# see https://github.com/realm/ci/packages/147854 for instructions to authenticate
+FROM docker.pkg.github.com/realm/ci/mongodb-realm-test-server
+
+RUN yum update -y \
+    && yum install -y \
+        curl \
+        wget
+
+RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+RUN chmod u+x jq
+RUN curl -sL https://rpm.nodesource.com/setup_12.x | bash -
+RUN yum install -y nodejs
+RUN npm install mongodb-stitch-cli
+RUN ln -s ./node_modules/.bin/stitch-cli stitch-cli
+
+COPY import_app.sh /app/
```

Comment:
I think this is superseded by https://github.com/realm/ci/tree/master/realm/docker/mongodb-realm - the base image already has importing capabilities.

Reply:
I tried it and it's very close, but importing custom apps isn't working. I committed a fix to your run script but I'll have to wait for the nightly job to publish a new docker image to test so I'll fix/simplify all this when that's working.

Reply:
https://github.com/realm/ci/runs/512551368?check_suite_focus=true just completed some 20 minutes ago. Thanks for the fix!

Reply:
After another fix, the docker image is working (tested locally) but Jenkins needs a way to authenticate to Github to pull the image. Can you give the ci Github user a token or is there a better way to do this?

Reply:
https://github.com/realm/ci/pull/283 should do the trick

Reply:
that works, thanks! 🎉